lib/srv/db/cloud/watchers/rds.go

```diff
@@ -98,7 +93,7 @@ func (f *rdsFetcher) Get(ctx context.Context) (types.Databases, error) {
 			f.log.Debugf("%v doesn't match selector.", database)
 		}
 	}
-	return result, nil
+	return result, trace.NewAggregate(rdsErr, auroraErr)
```

Comment:
personally, i don't like the idea returning partial result + error. so alternative is never turn any error for this function, just print warning when errors are found above. What do you think?

Reply:
Cool. I will add comment explaining why doing this. On a second look at this code tho, I feel the code might be cleaner if we split the current fetcher to two fetchers, one for rds instances, one for aurora clusters. That way we can just return error as before and not worry about aggregate. What do you think?